desc/compute/_omnigenity.py

```diff
@@ -306,20 +362,21 @@ def _zeta_B(params, transforms, profiles, data, **kwargs):
 
 
 @register_compute_fun(
-    name="sqrt(g)_B",
-    label="\\sqrt{g}_{B}",
+    name="sqrt(g)_Boozer_DESC",
```

Comment:
Do we have a way to deprecate these names and throw a warning or something? We should consider doing that to allow a changeover period

Reply:
I don't think we currently have a way. I guess the place to add it would be to add an alias here as sqrt(g)_B, then inside the compute function (not the `_compute` function), check if the requested name is in deprecated_names and throw a warning? should not be too hard.

Reply:
Added a deprecation warning